index.html

```diff
@@ -351,7 +351,15 @@ <h3>Defining language values</h3>
 	</details>
 	</div>
 
- 	<p>BCP 47 contains one RFC dedicated to the syntax and subtags of language tags, and another dedicated to how to match two or more subtags.  (This topic needs more detail, and may merit being a separate section.)</p>
+ 	<p>BCP 47 contains one RFC dedicated to the syntax and subtags of language tags, and another dedicated to how to match two or more subtags.</p>
+ 	
+ 	<aside class="issue"><p>The topic of matching language tags needs more detail, and may merit being a separate section.</p></aside>
+ 	
+ 	<div class="req" id="use_lstr">
+		<p class="advisement">Specifications SHOULD refer to the IANA Language Subtag Registry instead of providing lists of codes extracted from ISO 639, ISO 3166, or other standards.</p>
+ 	</div>
+ 	
+ 	<p>As part of BCP 47, IANA maintains the language subtag registry, which is a publicly available, machine-readable list of valid subtags for use in language tags. While this registry is based on underlying ISO standards, such as ISO 639 (languages) and ISO 3166 (regions), the list is actively maintained, stabilized, and comprehensive in ways that other lists found on the Internet may not be. Each of the subtag types is kept in sync with parent standards with the help and participation of those standards maintainers. These include the various parts of ISO 639 (639-1, 639-2, 639-3), the ISO 15924 script codes, and ISO 3166 and UN M.49 region codes. Extracting or making your own list of codes or referring to ones found elsewhere can lead to maintenance problems or confusion.</p>
```

Comment:
You could also add something like:
"Selecting one of the ISO or UN code lists will unnecessarily restrict the list of languages that can be described."

Reply:
Good callout. I did a more extensive edit, plus I added links to the "choosing a language tag" article, your subtag lookup tool, and the registry itself.